examples/demo_ot_detector.ipynb

```diff
@@ -0,0 +1,1403 @@
+       "      <td>Male</td>\n",
+       "      <td>United-States</td>\n",
+       "      <td>37-47</td>\n",
+       "      <td>10-11</td>\n",
+       "      <td>40-44</td>\n",
+       "      <td>7298-7978</td>\n",
+       "      <td>0</td>\n",
+       "      <td>1</td>\n",
+       "      <td>0.236226</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>?</td>\n",
+       "      <td>Some-college</td>\n",
+       "      <td>Never-married</td>\n",
+       "      <td>?</td>\n",
+       "      <td>Own-child</td>\n",
+       "      <td>White</td>\n",
+       "      <td>Female</td>\n",
+       "      <td>United-States</td>\n",
+       "      <td>17-27</td>\n",
+       "      <td>10-11</td>\n",
+       "      <td>1-39</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0.236226</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "    workclass      education       marital_status          occupation  \\\n",
+       "0     Private           11th        Never-married   Machine-op-inspct   \n",
+       "1     Private        HS-grad   Married-civ-spouse     Farming-fishing   \n",
+       "2   Local-gov     Assoc-acdm   Married-civ-spouse     Protective-serv   \n",
+       "3     Private   Some-college   Married-civ-spouse   Machine-op-inspct   \n",
+       "4           ?   Some-college        Never-married                   ?   \n",
+       "\n",
+       "  relationship    race      sex  native_country age_bin education_num_bin  \\\n",
+       "0    Own-child   Black     Male   United-States   17-27               1-8   \n",
+       "1      Husband   White     Male   United-States   37-47                 9   \n",
+       "2      Husband   White     Male   United-States   28-36             12-16   \n",
+       "3      Husband   Black     Male   United-States   37-47             10-11   \n",
+       "4    Own-child   White   Female   United-States   17-27             10-11   \n",
+       "\n",
+       "  hours_per_week_bin capital_gain_bin capital_loss_bin  observed  expectation  \n",
+       "0              40-44                0                0         0     0.236226  \n",
+       "1              45-99                0                0         0     0.236226  \n",
+       "2              40-44                0                0         1     0.236226  \n",
+       "3              40-44        7298-7978                0         1     0.236226  \n",
+       "4               1-39                0                0         0     0.236226  "
+      ]
+     },
+     "execution_count": 22,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "data = pd.read_csv('https://gist.githubusercontent.com/Viktour19/b690679802c431646d36f7e2dd117b9e/raw/d8f17bf25664bd2d9fa010750b9e451c4155dd61/adult_autostrat.csv')\n",
```

Comment:
why do you use this version of adult instead of the normal one?

Reply:
It was the one used in the MDSS example notebook - will change to use `load_preproc_data_adult`